src/applications/edu-benefits/utils/labels.jsx

```diff
@@ -144,16 +144,20 @@ export const directDepositWarning = (
     isn’t an option for Chapter 32 (VEAP) recipients). If you don’t have a bank
     account, you must get your payment through Direct Express Debit MasterCard.
     To request a Direct Express Debit MasterCard you must apply at{' '}
-    <a href="http://www.usdirectexpress.com" target="_blank">
+    <a
+      href="http://www.usdirectexpress.com"
+      target="_blank"
+      rel="noopener noreferrer"
+    >
       www.usdirectexpress.com
     </a>{' '}
     or by telephone at{' '}
-    <a href="tel:8003331795" target="_blank">
+    <a href="tel:8003331795" rel="noopener noreferrer" target="_blank">
```

Comment:
I don't know what to think of this, but leaving the rel attribute out here results in an error.

Reply:
It doesn't make sense to have tel links open in a new tab, I think you could remove the target attribute.